mailstore-connector/src/com/axonivy/connector/mailstore/MailStoreService.java

```diff
@@ -688,7 +688,7 @@ private static Properties getProperties() {
 				String propertyName = name.substring(propertiesPrefix.length());
 				String value = variable.value(); 
 				LOG.info("Setting additional property {0}: ''{1}''", propertyName, value);

```

Comment:
thank @anh-bolt a lot for your contribution!
I just want to get the idea for this line code, does it leak any privacy data to the ivy.log file?

Reply:
hi @nqhoan-axonivy, the possible property list are in the end of this link : https://javaee.github.io/javamail/docs/api/com/sun/mail/imap/package-summary.html
honestly, i'm not sure which property is privacy. this property 'mail.imap.proxy.password' ?